README.md

```diff
@@ -86,7 +60,8 @@ All the exception and log messages in this project are consistent with the follo
 When contributing to this project, please follow these guidelines.
 
 ## License
+
 ScalarDB is dual-licensed under both the Apache 2.0 License (found in the LICENSE file in the root directory) and a commercial license.
 You may select, at your option, one of the above-listed licenses.
 The commercial license includes several enterprise-grade features such as ScalarDB Server, management tools, and declarative query interfaces like GraphQL and SQL interfaces.
-Regarding the commercial license, please [contact us](https://scalar-labs.com/contact_us/) for more information.
+Regarding the commercial license, please [contact us](https://www.scalar-labs.com/ja/contact-jp) for more information.
```

Comment:
To give users the option of contacting either the US or Japan office, how about we provide the following page, which lets them choose?:

```suggestion
For more information about the commercial license, please [contact us](https://www.scalar-labs.com/contact).
```